ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -19,6 +19,16 @@
 # This file defines fan-coil unit types. Valid types must include a fan and a coil of some type.
 # They must also include some control scheme (e.g. ZTC, DSP, etc.).
 
+FCU_DFVSC_CHWZTC_HWZTC:
```

Comment:
After review, I believe this type should have all the points necessary to be modeled as the existing type FCU_DFSS_DFVSC_ZTC_CHWZTC_HWZT.

Reply:
thank you @ccquigley - that works perfectly. Closing this PR